src/hyperion/device_setup_plans/setup_panda.py

```diff
@@ -32,30 +35,37 @@ class PcapArm(Enum):
     DISARMED = "Disarm"
 
 
-def get_seq_table(
-    parameters: PandAGridScanParams,
-    exposure_distance_mm,
+def _get_seq_table(
+    parameters: PandAGridScanParams, exposure_distance_mm, time_between_steps_ms
 ) -> SeqTable:
     """
-    -Exposure distance is the distance travelled by the sample each time the detector is exposed: exposure time * sample velocity
-    -Setting a 'signal' means trigger PCAP internally and send signal to Eiger via physical panda output
-    -When we wait for the position to be greater/lower, give a safe distance (X_STEP_SIZE/2 * MM_TO_ENCODER counts) to ensure the final trigger point
-    is captured
+    Generate the sequencer table for the panda.
+
+    - Sending a 'trigger' means trigger PCAP internally and send signal to Eiger via physical panda output
+
     SEQUENCER TABLE:
-        1:Wait for physical trigger from motion script to mark start of scan / change of direction
-        2:Wait for POSA (X2) to be greater than X_START, then
-            send a signal out every (minimum eiger exposure time + eiger dead time)
-        3:Wait for POSA (X2) to be greater than X_START + X_STEP_SIZE + a safe distance for the final trigger, then cut out the signal
-        4:Wait for physical trigger from motion script to mark change of direction
-        5:Wait for POSA (X2) to be less than X_START + X_STEP_SIZE + exposure distance, then
-            send a signal out every (minimum eiger exposure time + eiger dead time)
-        6:Wait for POSA (X2) to be less than (X_START - safe distance + exposure distance), then cut out signal
-        7:Go back to step one.
+
+        1. Wait for physical trigger from motion script to mark start of scan / change of direction
+        2. Wait for POSA (X2) to be greater than X_START and send 1 trigger
+        3. Send out the remaining x_steps - 1 triggers every time_between_steps_ms
```

Comment:
Can we combine these two steps? Eg the second row could look like
repeats = num_pulses, trigger = POSA>POSITION, TIME1=exposure_time, TIME2=`time_between_x_steps`, outa1 = 1, outa2 = 0

Reply:
Same for 5 and 6